src/main/kotlin/no/nav/familie/ks/sak/kjerne/vilkårsvurdering/domene/VilkårResultat.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.familie.ks.sak.kjerne.vilkårsvurdering.domene
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import no.nav.familie.ks.sak.common.entitet.BaseEntitet
+import no.nav.familie.ks.sak.common.util.StringListConverter
+import no.nav.familie.ks.sak.kjerne.vedtak.Standardbegrunnelse
+import no.nav.familie.ks.sak.kjerne.vedtak.StandardbegrunnelseListConverter
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Convert
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.SequenceGenerator
+import javax.persistence.Table
+
+@Entity(name = "VilkårResultat")
+@Table(name = "vilkar_resultat")
+class VilkårResultat(
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "vilkar_resultat_seq_generator")
+    @SequenceGenerator(
+        name = "vilkar_resultat_seq_generator",
+        sequenceName = "vilkar_resultat_seq",
+        allocationSize = 50
+    )
+    val id: Long = 0,
+
+    // Denne må være nullable=true slik at man kan slette vilkår fra person resultat
```

Comment:
```suggestion
    // Denne må være nullable slik at man kan slette vilkår fra person resultat
```
Kanskje vi bare kan fjerne =true delen slik at det ikke misforstås med @Column(nullable=true)

Reply:
Prøver meg på det!